src/containers/profile/Profile.js

```diff
@@ -13,46 +12,29 @@ export default function Profile() {
   function setProfileFunction(array) {
     setrepo(array);
   }
-  function getProfileData() {
-    const client = new ApolloClient({
-      uri: "https://api.github.com/graphql",
-      request: operation => {
-        operation.setContext({
-          headers: {
-            authorization: `Bearer ${openSource.githubConvertedToken}`
-          }
-        });
-      }
-    });
 
-    client
-      .query({
-        query: gql`
-      {
-        user(login:"${openSource.githubUserName}") { 
-          name
-          bio
-          isHireable
-          avatarUrl
-          location
-        }
-    }
-      `
-      })
-      .then(result => {
-        setProfileFunction(result.data.user);
-      })
-      .catch(function (error) {
-        console.log(error);
-        setProfileFunction("Error");
-        console.log(
-          "Because of this Error Contact Section is Showed instead of Profile"
-        );
-        openSource.showGithubProfile = "false";
-      });
-  }
   useEffect(() => {
     if (openSource.showGithubProfile === "true") {
+      const getProfileData = () => {
+        fetch("/profile.json")
+          .then(result => {
+            if (result.ok) {
+              return result.json();
+            }
+            console.error(result);
+          })
+          .then(response => {
+            setProfileFunction(response.data.user);
+          })
+          .catch(function (error) {
+            setProfileFunction("Error");
+            console.log(
+              "Because of this Error Contact Section is Showed instead of Profile"
```

Comment:
nit: We can maybe change this to something like 'Because of this error, contact section has reverted to default' or something since both the variants are for contact section.

Reply:
yeah, I didn't change these actually, I just copy-pasted how it was previously, let me change it.